tests/core/selectors_test.py

```diff
@@ -20,12 +20,31 @@
 
 import collections
 import dataclasses
-from typing import Any
+from typing import Any, Iterable
 
 from absl.testing import absltest
 import jax
 from penzai import pz
 import penzai.core.selectors
+import pytest
+
+
+@pytest.mark.parametrize(
+  "input_indices, shift, expected_output",
+  [
+    ((), 1, ()),
+    ([0, 3, -2], len(range(6)), (0, 3, 4)),
+  ]
+)
+def test_shift_negative_indices(
+  input_indices: Iterable[int],
+  shift: int,
+  expected_output: tuple[int, ...],
+):
+  assert (
+    penzai.core.selectors._shift_negative_indices(input_indices, shift=shift)
+    == expected_output
+  )
```

Comment:
Would you mind adding this as an absltest-style test (as a method inside SelectorsTest) instead of using pytest here? The rest of the tests do not use pytest yet and I'd prefer to be consistent here for now.